core/server/api/v2/utils/serializers/input/posts.js

```diff
@@ -89,6 +89,7 @@ module.exports = {
          * @deprecated: `author`, might be removed in Ghost 3.0
          */
         if (frame.data.posts[0].hasOwnProperty('author')) {
+            // @TODO: remove + deny
```

Comment:
If we would use a JSON schema, we could control which attrs are allowed. But we don't use one yet.